README.md

```diff
@@ -1,18 +1,59 @@
 [comment]: # ( Copyright Contributors to the Open Cluster Management project )
 
-# Configuration Policy Controller [![KinD tests](https://github.com/open-cluster-management/config-policy-controller/actions/workflows/kind.yml/badge.svg?branch=main&event=push)](https://github.com/open-cluster-management/config-policy-controller/actions/workflows/kind.yml)
+# Configuration Policy Controller
+
```

Comment:
can you rename `Red Hat Advanced Cluster Management - Governance - Configuration Policy Controller` to `Open Cluster Management - Configuration Policy Controller`?